src/main/scala/exercises/04_Options.scala

```diff
@@ -44,6 +52,6 @@ object Options {
    * to convert a possible numeric String (e.g. Some("12")) to an integer
    */
   def totalPeopleInRooms(rooms: Map[Int, Option[String]]): Int = {
-    error("Fix me")
+	rooms.foldLeft(0)((acc, kv) => acc + peopleInRoom(kv._2))
```

Comment:
I used map and sum to compute the total number of people, but it's interesting how you used foldLeft for this task.